### PR TITLE
Add CentOS 7.7 (aarch64)

### PR DESCRIPTION
**Pull request by akopytov**
Port CentOS 7.7 related changes from master.

### CODE REVIEW

File: centos/7/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,8 @@
-FROM centos/aarch64:latest
+FROM centos:7
```
Comment (Totktonada) [inline centos/7/Dockerfile:1]: Will not this pull an amd64 image?

Comment (akopytov) [reply centos/7/Dockerfile:1]: CentOS uses multi-arch images now: https://hub.docker.com/layers/centos/library/centos/7/images/sha256-1f832b4e3b9ddf67fd77831cdfb591ce5e968548a01581672e5f6b32ce1212fe?context=explore

Comment (Totktonada) [reply centos/7/Dockerfile:1]: Thanks for the clarification!

**Review by Totktonada — APPROVED** [review]
LGTM.

**Pull request merged**